backports/py-pysdl2-dll/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Contributor: Florent Ferreri <florent@seqsense.com>
+# Maintainer:
+pkgname=py-pysdl2-dll
+_pkgname=pysdl2-dll
+pkgver=2.24.1
+pkgrel=0
+pkgdesc="A package that bundles the SDL2 libraries for PySDL2 so they don't have to be installed manually"
+url="https://github.com/a-hurst/pysdl2-dll"
+arch="all"
+license="MPL-2.0"
+depends="sdl2-dev"
```

Comment:
Is it correct to have runtime dependency to -dev package?
Usually it's like `depends="foo" makedepends="foo-dev"`